Datasets/data-connect-dataset-outlookmeetingactivity.md

```diff
@@ -0,0 +1,92 @@
+
+## Scenarios
+
+The following are business scenarios that you can answer with this dataset:
+
+- Analyze patterns of communication of employees 
+- There can be an Outlook meeting effectiveness dashboard based on these signals.
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- How many meetings were interacted with by users during a time range? 
+- What is the relative number of meetings compared to last month?  
+- How many meeting invites were accepted/declined etc?  
+- How often do employees attend meetings? 
+
+## Joining with other datasets
+
+The OutlookMeetingActivity_v0 dataset can be joined with other activities datasets, to get insights from how employees in the organization interact through outlook and teams. 
+
+## Definitions
+
+- **Signal:**  A Substrate signal is a high-value event that captures a user's behavior. A signal can be either active (like joining a meeting) or passive (like a location change).
+
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| SignalType | string | Name of the signal. | No | None |
+| Id | string | The unique identifier of the Signal. | No | None |
+| StartTime | datetime | The date and time when action happens. | No | None |
+| CreationTime | datetime | The date and time when the signal was created. | Yes | Date |
+| EndTime | datetime | The date and time when action ends. If a signal is a point-in-time action, it is same as StartTime. If the action lasts a time period, this is the action end time captured on client. | No | None|
+| ItemType | string | The type of item being acted on. | No | None |
+| AppName | string | The end user client name being used to generate the signal, E.g. "Unknown", "OutlookMobile", "OWA", "OutlookDesktop", etc. | No | None |
+| AadTenantId | string | AadTenantId of the actor (signal generator). | No | None |
+| ClientSessionId | string | Unique session id for the user, among the properties defined by the signal producer. | No | None |
+| MailboxGuid | string | Unique mailbox id for the user, among the properties defined by the signal producer. | No | None |
+| TenantName | string | The tenant name, among the properties defined by the signal producer. | No | None|
+| OId | string | UserId, among the properties defined by the signal producer. | No | None |
+| puser | String | User id. | No  | None |
```

Comment:
nit: for datatypes of puser and ptenant, it is "String" while rest of others are "string". 
Please maintain case parity across all 5 datasets for the puser. ptenant in both table + json representation with rest of the columns